### PR TITLE
Add detailed info about Device Farm

### PR DESCRIPTION
**Pull request by artemnikitin**
Comments and remarks are welcome because I'm not a native English speaker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
 - Supports iOS and Android (including Kindle Fire) devices, as well as the mobile web.
 - Supports remote device access in order to allow for interactive testing/debugging.
 
-🚧 [*Please help expand this incomplete section.*](CONTRIBUTING.md)
+### Device Farm Tips
+- [AWS Mobile blog](https://aws.amazon.com/blogs/mobile/) contains several examples of Device Farm usage for testing.
+- Device Farm offers a free trial for users who want to evaluate their service.
+- Device Farm offers two pricing model depends on a client use case. Pay per device minute is useful for small usage levels or for situations where it's hard to predict an amount of usage. Unmetered plans are useful in situations where active usage expected from beginning.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "Device Farm offers two pricing model: “pay per device minute”, which is useful for small usage levels or for situations where it‘s hard to predict usage amount and “unmetered plans”, which are useful in situations where active usage is expected from the beginning."

File: README.md [hunk 2]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
...
+
+### Device Farm Gotchas and Limitations
+- ❗Devices don't have a SIM card and therefore can't be used for testing SIM card related features.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit: use Unicode apostrophes (‘).

Comment (ThanosBaskous) [inline README.md]: Punctuation suggestion: "SIM card related" -> "SIM card-related"

File: README.md [hunk 3]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
...
+- 🔸Device Farm supports testing for most popular languages\frameworks, but not for all. An actual list of supported frameworks and languages presented on [this page](http://docs.aws.amazon.com/devicefarm/latest/developerguide/test-types-overview.html).
```
Comment (ThanosBaskous) [inline README.md]: Conjunction: use front slash ("languages/frameworks")

Comment (ThanosBaskous) [inline README.md]: Missing word: "and languages presented on" -> "and languages is presented on"

File: README.md [hunk 4]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
...
+- 🔸AWS provide an API and CLI for Device Farm, but API is quite a low level and may require developing additional tools or scripts on top of it.
```
Comment (ThanosBaskous) [inline README.md]: Suggestion: No need to repeat that AWS is providing things - it's implied, I think. "🔸The API and CLI for Device Farm may be too low level for many use cases and could require the development of additional tools or scripts."

Comment (ThanosBaskous) [inline README.md]: Non-blocking: Any third party tools that are worth mentioning?

File: README.md [hunk 5]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
...
+- 🔸AWS provide several tools and plugins for Device Farm, however, it doesn't cover all cases or platforms. It may require developing specific tools or plugins to support specific requirements.
```
Comment (ThanosBaskous) [inline README.md]: Mostly redundant - suggest removing.

File: README.md [hunk 6]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
...
+- ❗In general, Device Farm doesn't have Android devices from Chinese companies like Huawei, Meizu, Lenovo, etc. An actual list of supported devices located [here](https://aws.amazon.com/device-farm/device-list/).
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit: use Unicode apostrophes (‘).

**Comment by artemnikitin** [comment]
Fixed all suggestions.

File: README.md [hunk 7]
```diff
@@ -1374,7 +1374,17 @@ Device Farm
 - Supports iOS and Android (including Kindle Fire) devices, as well as the mobile web.
 - Supports remote device access in order to allow for interactive testing/debugging.
 
-🚧 [*Please help expand this incomplete section.*](CONTRIBUTING.md)
+### Device Farm Tips
+- [AWS Mobile blog](https://aws.amazon.com/blogs/mobile/) contains several examples of Device Farm usage for testing.
+- Device Farm offers a free trial for users who want to evaluate their service.
+- Device Farm offers two pricing model: “pay per device minute”, which is useful for small usage levels or for situations where it‘s hard to predict usage amount and “unmetered plans”, which are useful in situations where active usage is expected from the beginning.
```
Comment (ThanosBaskous) [inline README.md]: I think I told you to use the wrong apostrophe throughout (‘ vs ’) - my mistake, sorry about that @artemnikitin! No need to fix in this PR - I'll fix after merge.

**Comment by jlevy** [comment]
👍 Thanks for the great additions! Keep 'em coming... :)

**Pull request merged**